### PR TITLE
Sponsor Button Alignment

### LINKED ISSUE #270: "Become a sponsor" button alignment
In the mobile view, the sponsor button is hardcoded to stay at x=5 and y=5, which makes it align to left. It should be aligned to the center to look good and to be noticed easily.

Screenshot - 
Screenshot (138)

### If this isn't done purposely I would like to work on it.

### PR DESCRIPTION
**Pull request by Yashank18**
Fixes #270
 ### Files Changed -  Index.html 
### Before - 
Screenshot (143) 
Screenshot (142) 
### After - 
Screenshot (146)
 Screenshot (140) 
Screenshot (139)

### CODE REVIEW

**Comment by Yashank18** [comment]
Please review this PR
@jancborchardt  @vinitshahdeo @RDIL 
Thank you

File: index.html [hunk 1]
```diff
@@ -73,9 +73,9 @@ <h2>Supporters</h2>
               <img src="images/supporters/redhat.svg" height="50px" width="auto" alt="RedHat Logo"/>
               <a title="Stickermule" href="http://stickermule.com/supports/opensourcedesign19-sponsorship" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
               <img src="images/supporters/wikimedia-germany.svg" height="50px" width="auto" alt="Wikimedia Germany Logo"/>
+              <center><object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=50&width=110"></object></center>
```
Comment (vinitshahdeo) [inline index.html:76]: @Yashank18 `<center> .. </center>` tag is deprecated now. I encourage you not to use this.

Please **play around CSS to center align this.** One method of centering is to set the `margin-left` and `margin-right` properties of the element to `auto`, and then set the parent element’s `text-align` property to center. This guarantees that the element will be centered in all modern browsers.

Comment (Yashank18) [reply index.html]: @vinitshahdeo Thank you for reviewing. It was my bad. I just realized its parent element already has a center align in CSS so I just made changes, These will solve the issue and it increases the size, as well as this button, wasn't that noticeable.

**Comment by Yashank18** [comment]
@RDIL it was my bad. I just realized its parent class already has a center-aligned CSS property. So I just made changes, Please Check 
Thank you

**Pull request merged**